client.go

```diff
@@ -16,7 +16,13 @@ type Client interface {
 	Push() (string, error)
 	// Run runs the specified deploy, returning the port it is listening on.
 	Run(deployId string) (int, error)
+
+	// Sets active deploy by port
```

Comment:
Can be a TODO but it seems that SetActiveByPort() and SetActiveById() are better names than the current ones.